client/src/components/history/history-item.tsx

```diff
@@ -73,15 +77,78 @@ export function ChatItem(props: {
   }
 
   function LinkRenderer(props: { href: string; children: React.ReactNode }) {
-    return (
-      <h3>
-        <a href={props.href} target="_blank" rel="noreferrer">
-          {props.children}
-        </a>
-      </h3>
+    const linkAnswer =
+      props.href.length > 30 ? props.href.slice(0, 30) : props.href;
+    return message.text.includes(prefix) ? (
+      <a
+        href="#"
+        rel="noreferrer"
+        onClick={sentQuestion}
+        data-cy={`question-link-${i}`}
+      >
+        {props.children}
+      </a>
+    ) : (
+      <a
+        href={props.href}
+        target="_blank"
+        rel="noreferrer"
+        onClick={sentQuestion}
+      >
+        {linkAnswer}
+      </a>
     );
   }
 
+  async function sentQuestion() {
+    const REGEX_ASK_LINK = /ask:\/\/display=(.*)/;
```

Comment:
let's discuss this. There are two possibilities:

1. everything after `ask://` is the display text (in which case we don't need `display=`
or
2. we're trying to support key value pairs to allow for different params in the future, eg..
`ask://display=click+here&another-param=some-value`

I favor option 1 for now if `display` is the only thing we have in there amyway, so let's make it

```typescript
const REGEX_ASK_LINK = /^ask:\/\/(.*)$/;
```

Reply:
I also prefer option 1, and it will be better and simpler.